Misc/NEWS.d/next/Core and Builtins/2018-12-03-21-20-24.bpo-35357.rhhoiC.rst

```diff
@@ -0,0 +1,3 @@
+Internal attributes' names of unittest.mock._Call and
+unittest.mock.MagicProxy (name, parent & from_kall) are now prefixed by
+_mock_
```

Comment:
This should describe the why of the change not just the what (in this case, because of the potential name clash)

Reply:
Done.